src/appmixer/utils/ai/CreateTranscription/CreateTranscription.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
             knownLength: fileInfo.length
         });
         formData.append('model', context.config.CreateTranscriptionModel || 'whisper-1');
-        formData.append('response_format', 'json');
+        formData.append('response_format', responseFormat || 'text');
```

Comment:
`json` response type is not supported anymore? it's a breaking change then. Unless the `json` response was a faulty response type and component wasn't working anyway.

Reply:
It's not a breaking change. The output of the component goes to `text` no matter what the format is. See the change in the line below from:

```
context.sendJson(data, 'out');
```
to:
```
context.sendJson({ text: data }, 'out');
```